src/backend.ts

```diff
@@ -25,18 +22,62 @@ export class CapacitorBackend extends BaseBackend<BrowserOptions> {
     super(_options);
     this._browserBackend = new BrowserBackend(_options);
 
-    if (this._isCapacitor() && _options.enableNative !== false) {
-      void this._startWithOptions();
+    void this._startWithOptions();
+  }
+
+  /**
+   * If native client is available it will trigger a native crash.
+   * Use this only for testing purposes.
+   */
+  public nativeCrash(): void {
+    if (this._options.enableNative) {
+      NATIVE.crash();
+    }
+  }
+
+  /**
+   * @inheritDoc
+   */
+  protected _setupCapacitorTransport(): Transport | NoopTransport {
+    if (!this._options.dsn) {
+      // We return the noop transport here in case there is no Dsn.
+      return new NoopTransport();
+    }
+
+    const transportOptions = {
+      ...this._options.transportOptions,
+      dsn: this._options.dsn,
+    };
+
+    if (this._options.transport) {
+      return new this._options.transport(transportOptions);
+    }
+
+    if (this._isNativeTransportAvailable()) {
+      return new NativeTransport();
     }
+
+    return new Transports.FetchTransport(transportOptions);
   }
 
   /**
    * Has Capacitor on window?
    */
-  private _isCapacitor(): boolean {
+  // private _isCapacitor(): boolean {
+  //   return (
+  //     getGlobalObject<any>().capacitor !== undefined ||
+  //     getGlobalObject<any>().Capacitor !== undefined
+  //   );
+  // }
```

Comment:
Do we not need this at all? Or are we saving this for later?

Reply:
@joshdholtz I'm potentially saving this for later. I'm uncertain right now whether I'll need it yet.